### PR TITLE
Test plan for LAG on Distributed VOQ System

### PR DESCRIPTION
**Pull request by oxygen980**
### Description of PR
The PR is to introduced LAG on VOQ 

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?

This is the test plan for LAG on SONIC Distributed VOQ System, as described in the [VOQ LAG HLD](https://github.com/Azure/SONiC/blob/2e05c6b8ac570fd237484a18e732a58eec004b9c/doc/voq/lag_hld.md)

The PR covered in this test plan is [Distributed VOQ LAG HLD PR 697](https://github.com/Azure/SONiC/pull/697/files#diff-77ea0c16b4ae9885fa0e388e81f6343c6bda0f24b999f93e64fcee8467df63fc)

The scope of this test plan is as follows:
* Check for unique SYSTEM_LAG_ID per LAG in a chassis with multiple line cards
* Check if LAG is learned by local and remote ASIC

This test plan does not cover functioning of LAG as an IP/Layer3 endpoint as that is outside the scope of the code PR.

#### How did you do it?
Based on the code changes based on the associated PR
#### Any platform specific information?
This is valid only on VOQ systems.

### CODE REVIEW

**Comment by oxygen980** [comment]
retest vs image please

**Comment by anshuv-mfst** [comment]
@eswaranb - please add Arista reviewers

**Comment by oxygen980** [comment]
retest vs image please

**Comment by oxygen980** [comment]
retest vs image please

**Comment by oxygen980** [comment]
retest vs image please

**Comment by oxygen980** [comment]
retest vs image please

**Comment by shubav** [comment]
@yxieca, Could you help merge this test plan? The failures in the test case are unrelated to this commit. Thanks.

**Comment by anshuv-mfst** [comment]
@yxieca @wangxin , @saravanansv - could you please take a look, thanks.

File: docs/testplan/chassis-lag-test-plan.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+lag id should exist in ASIC DB, repeat the same step on all ASIC on other host.
+```
+
+## Test Case 2. Dynamic deletion of LAG via CLI
+### Test Objective
+
+Verify that when a LAG is dynamically deleted via CLI on an ASIC, it is removed in remote ASIC_DB.
+### Test Steps
+* Delete an existing LAG from any ASIC in the OC topology
```
Comment (wangxin) [inline docs/testplan/chassis-lag-test-plan.md:127]: What is `OC topology`?

Comment (oxygen980) [reply docs/testplan/chassis-lag-test-plan.md]: OC topology was to refer Open community topology, I have removed 'OC topology'.

**Review by wangxin — APPROVED** [review]

**Pull request merged**